dashboard/app/views/followers/student_user_new.html.haml

```diff
@@ -1,3 +1,6 @@
+- content_for(:head) do
+  %meta{name: "robots", content: "noindex"}
```

Comment:
```suggestion
  %meta{name: "robots", content: "noindex"}/
```
`<meta>` is an [empty element](https://developer.mozilla.org/en-US/docs/Glossary/Empty_element). Adding the [trailing slash](http://haml.info/docs/yardoc/file.REFERENCE.html#empty-void-tags-) tells HAML to generate a self-closing tag or no closing tag as appropriate for the current format, to ensure valid markup.